spec/models/ingredient_mention_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe IngredientMention, type: :model do
+  subject do
+    build(:ingredient_mention)
+  end
+
+  describe 'when observing relationships' do
+    context 'with movies' do
+      it 'is valid' do
```

Comment:
Probably want to change 'is valid' to something like 'belongs to a movie' for more clarity